### ISSUE TITLE
Traefik manipulates original value in Content-Type HTTP header.

### ISSUE THREAD

**Issue by AviMualem** [containous/traefik#3131]
### Do you want to request a *feature* or report a *bug*?
Bug

### What did you do?
I tried to load a swagger site which documents my web api and noticed that Traefik manipulates the Content-Type HTTP header in the response for various css files.
While invoking the same web request without Traefik in the middle (going directly to the server) i noticed the difference in the mentioned header.

#### Request with Traefik:

http://localhost:5555/alerts/swagger/css/screen.css

```
###request headers:
Accept: text/css,*/*;q=0.1
Accept-Encoding: gzip, deflate
Accept-Language: en-US,en;q=0.5
Cache-Control: no-cache
Connection: keep-alive
Cookie: ai_user=FZ4cS|2017-11-02T14:47:58.633Z
Host: localhost:5555
Pragma: no-cache
Referer: http://localhost:5555/alerts/swagger/
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:59.0) Gecko/20100101 Firefox/59.0
```
```
### response headers:
HTTP/1.1 200 OK
##### Content-Type: text/plain; charset=utf-8
Date: Wed, 04 Apr 2018 15:22:12 GMT
Transfer-Encoding: chunked
```

#### Request without Traefik:
http://localhost:30020/swagger/css/screen.css

```
### request headers:
Accept: text/css,*/*;q=0.1
Accept-Encoding: gzip, deflate
Accept-Language: en-US,en;q=0.5
Cache-Control: no-cache
Connection: keep-alive
Cookie: ai_user=FZ4cS|2017-11-02T14:47:58.633Z
Host: localhost:30020
Pragma: no-cache
Referer: http://localhost:30020/swagger/
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:59.0) Gecko/20100101 Firefox/59.0
```

```
### response headers:
HTTP/1.1 200 OK
Accept-Ranges: bytes
Content-Length: 43644
##### Content-Type: text/css
Date: Wed, 04 Apr 2018 15:23:42 GMT
ETag: "24c85a5ed12703fc"
Last-Modified: Fri, 31 Dec 9999 23:59:59 GMT
Server: Microsoft-HTTPAPI/2.0
```

### What did you expect to see?
i expected to the same response headers i got when i accessed the server directly. 

### What did you see instead?
the content-type header value was text/plain; charset=utf-8 instead of text/css which made the swagger page to show its content and in a different way.

### Output of `traefik version`: (_What version of Traefik are you using?_)
1.5.2

**Comment by ldez** [comment]
Thanks for your interest in Traefik !

Could you edit the description of the issue to add your configuration (TOM, docker-compose, ...).

**Comment by sickill** [comment]
I'm experiencing the same issue.

From I observed it, traefik add `Content-Type: text/plain` to responses from backend which don't included this header. Backend obviously doesn't do a good job, but anyway, this breaks some JS apps (json and css files with `text/plain` don't get properly handle by Chrome).

**Comment by ldez** [comment]
@sickill could you provide your configuration and a docker-compose to reproduce your issue?

**Comment by sickill** [comment]
@ldez here is the config:

docker-compose.yml:

```yaml
version: '2'
services:
  traefik:
    image: traefik
    volumes:
    - traefik_acme:/mnt:nocopy
    - /home/rancher/traefik.toml:/traefik.toml
    ports:
    - 80:80/tcp
    - 8080:8080/tcp
    - 443:443/tcp
    command:
    - --api
    - --accessLog
    - --rancher
    - --rancher.exposedbydefault=false
    - --rancher.metadata=true
    - --logLevel=INFO
    - --entryPoints=Name:http Address::80 Redirect.EntryPoint:https
    - --entryPoints=Name:https Address::443 TLS
    - --defaultentrypoints=http,https
    - --acme
    - --acme.email=foo@example.com
    - --acme.storage=/mnt/acme.json
    - --acme.entrypoint=https
    - --acme.onHostRule=true
    - --acme.httpChallenge.entryPoint=http
    - --acme.acmeLogging=true
```

traefik.toml:

```toml
[file]
[backends.example.servers.default]
url = "https://some.herokuapp.com"

[frontends.example]
entryPoints = ["https"]
backend = "example"

[frontends.example.routes.route0]
rule = "Host:some.host;PathPrefix:/app"

[frontends.example.headers.customrequestheaders]
Host = "some.herokuapp.com"
```

There are other frontends/backends enabled, but they're defined via Rancher labels, and the services for them are defined in separate docker-compose files.

**Comment by dbjpanda** [comment]
I am facing the same issue as well. Any fix or workaround for the issue.

**Comment by Ullaakut** [comment]
Could someone who has this issue provide their full configuration? I am unable to reproduce it so I might be missing something from what @sickill has shared.

**Comment by traefiker** [comment]
Hi! I'm Træfiker :robot: the bot in charge of tidying up the issues.

I have to close this one because of its lack of activity :disappointed:

Feel free to re-open it or join our [Slack workspace](https://traefik.herokuapp.com/) for more [community #support](https://traefik.slack.com/messages/C0CDT22PJ/).

**Comment by traefiker** [comment]
Closed by #6097.
